terraform/aws/modules/atlantis/atlantis.tf

```diff
@@ -5,10 +5,11 @@ resource "kubernetes_namespace" "atlantis" {
 }
 
 resource "helm_release" "atlantis" {
-  name      = "atlantis"
-  chart     = "runatlantis/atlantis"
-  namespace = "atlantis"
-  version   = var.atlantis_chart_version
+  name       = "atlantis"
+  chart      = "atlantis"
+  repository = data.helm_repository.atlantis.url
+  namespace  = kubernetes_namespace.atlantis.metadata.name
```

Comment:
@stafot you should set 
```
repository = "https://github.com/runatlantis/helm-charts
chart = "atlantis"
```
The data helm repo is deprecated.

Reply:
@stylianosrigas Thanks. On it.